app/controllers/authentication.js

```diff
@@ -20,9 +20,22 @@ class AuthenticationController {
   constructor() {
     this.config = {
       GOOGLE_SECRET: nconf.get('google_secret'),
-      TOKEN_SECRET: nconf.get('webtoken'),
+      TOKEN_SECRET: nconf.get('webtoken')
```

Comment:
Trailing comma is an eslint preference, personally I do not like it but that is why it exists.